fitbenchmarking/fitbenchmarking
# Remove headers

Modified license to STFC's preferred license: BSD 3 Clause Licence

Fixes #201 

#### Testing Instructions

1. Check all old headers remove
2. check that BSD 3 Clause Licence has been added correctly
